docs/python_docs/python/tutorials/packages/np/np-vs-numpy.md

```diff
@@ -91,16 +91,20 @@ b = a[0]
 b.ndim, b.size, isinstance(b, np.ndarray)
 ```
 
-## Save
+## Save and load
 
-The `save` method in `mxnet.np` saves data into a binary format that's not compatible with NumPy format. For example, it contains the device information. (TODO, needs more discussion here.) 
+Users should call the official NumPy `save` and `load` methods respectively to save and load arrays.
 
 ```{.python .input}
 a = np.array(1, ctx=gpu)
-npx.save('a', a)
-npx.load('a')
+onp.save('a', a)
+onp.load('a.npy', a)
 ```
 
+For advanced use-cases, MXNet further provides `npx.save` and `npx.load`, which
+can save and load a dictionary of arrays to the `.npz` format such that it can
```

Comment:
should we follow the savez naming convention?

Reply:
Added `npx.savez` that can save multiple arrays (dense or sparse) to the npz format. Updated `npx.save` to only support saving a single dense array to `.npy`.

I'm keeping them in the `npx` namespace as we do not support pickling objects etc.

WDYT?